examples/tf_vars/multiple_notifiers.tfvars.example

```diff
@@ -10,18 +10,18 @@ aws_region  = "us-west-2"
 global_prefix = "CHANGEME"  # prefix used for all ggcanary resources
 
 
-SendGrid_notifier = {
-  enabled = true
-  parameters = {
-    SOURCE_EMAIL_ADDRESS = "canary@my_domain.org"  # email address to send emails from
-    DEST_EMAIL_ADDRESS   = "security_email@my_domain.org"  # email address receiving alerts
-    API_KEY              = "SG.XXXX"  # Account API key
-  }
-}
+SendGrid_notifiers = [{
+  SOURCE_EMAIL_ADDRESS = "canary@my_domain.org"  # email address to send emails from
```

Comment:
In other examples, it is no more in capital letters ? Why is it still the case in this example file ?

Reply:
Well spot, missed this one.